docs/en/authorization_endpoint.rst

```diff
@@ -195,57 +172,35 @@ When the **scope** parameter is used, the following values are supported:
     - *email*;
     - *email_verified*.
 
-The attributes requested by the parameter **scope** are available both in the ID Token and in the response to the Userinfo Endpoint.
+.. admonition:: |spid-icon|
 
-.. note::
-   The parameter **scope** MAY contain one or more values, with single spaces as separators. For example, using both profile and email in the scope parameter returns the Minimum eIDAS Dataset and the email.
+  The attributes requested by the parameter **scope** are available in the response to the *userinfo endpoint*.
+  
+.. admonition:: |cieid-icon|
 
-In case of requests of single user-attributes or specific combinations of them, the RP MAY use the parameter **claims**. 
-For the definition of the parameter **claims** and its usage modes for requesting the user attributes, please refer to `OpenID.Core#ClaimsParameter`_. 
+  The attributes requested by the parameter **scope** are available both in the ID Token and in the response to the *userinfo endpoint*.
+  
+.. note::
 
-.. warning::
-    - Only for CIE id: In the object *id_token* of the parameter **claims**, it is possible to request only the eIDAS Minimum Dataset. The other user attributes MUST be requested in the object *userinfo* of the parameter **claims**. Moreover, the user attributes requested in the *id_token* object are also available at the *userinfo endpoint*.  
+   The parameter **scope** MAY contain one or more values, with single spaces as separators. For example, using both profile and email in the scope parameter returns the Minimum eIDAS Dataset and the email.
+   In case of requests of single user-attributes or specific combinations of them, the RP MAY use the parameter **claims**.
+   For the definition of the parameter **claims** and its usage modes for requesting the user attributes, please refer to `OpenID.Core#ClaimsParameter`_.
 
 .. warning::
-    - If the parameter **claims** is not present or has no value and the only scope openid has been requested, only the claim *sub* is returned
-      in the response to the Userinfo Endpoint. 
-
+  If the parameter **claims** is not present or has no value and the only scope openid has been requested, only the claim *sub* is returned in the response to the *userinfo endpoint*.
+  
+.. admonition:: |spid-icon|
 
```

Comment:
Only for SPID: 
Only if the RP metadata contains id_token_encrypted_response_alg and id_token_encrypted_response_enc and their value is not 'none', the user attributes requested by the id_token object into the **claims** parameter will be available into the id_token from the token endpoint, added to any claims that are being requested using scope values.

The user attributes requested by the userinfo object into the **claims** parameter will be available only into the response from the userinfo endpoint, added to any Claims that are being requested using scope values.

@nunzionapoli please check if this is compliant to OIDC Core.

Reply:
here my proposal

https://github.com/italia/spid-cie-oidc-docs/issues/82

so if you agree we may merge it as it is and then create a separate PR for that